docs/HowTo/Get-Started/Register-Validators.md

```diff
@@ -1,4 +1,4 @@
-description: Register validators     
+description: Register validators
 <!--- END of page meta data -->
```

Comment:
You'll need to wrap the meta data in "---". For example,
---
description: Register validators
---

Reply:
Done.